### PR TITLE
Make return same fact for 3 different languages #1419

### PR DESCRIPTION
**Pull request by vorfolomey**
#1419

### CODE REVIEW

File: core/src/main/java/greencity/service/impl/FactOfTheDayServiceImpl.java [hunk 1]
```diff
@@ -27,14 +34,17 @@
...
+@Service
+@Scope(proxyMode = ScopedProxyMode.TARGET_CLASS)
 public class FactOfTheDayServiceImpl implements FactOfTheDayService {
     private final FactOfTheDayRepo factOfTheDayRepo;
     private final ModelMapper modelMapper;
     private final LanguageService languageService;
     private final FactOfTheDayTranslationService factOfTheDayTranslationService;
     private final PlaceCommentServiceImpl placeCommentServiceImpl;
+    @Resource
+    private FactOfTheDayService self;
```
Comment (antonbakalets) [inline core/src/main/java/greencity/service/impl/FactOfTheDayServiceImpl.java:47]: What is the problem you are trying to solve here? Why not use `this` instead?

Comment (vorfolomey) [reply core/src/main/java/greencity/service/impl/FactOfTheDayServiceImpl.java:47]: https://stackoverflow.com/questions/16899604/spring-cache-cacheable-not-working-while-calling-from-another-method-of-the-s

Comment (vorfolomey) [reply core/src/main/java/greencity/service/impl/FactOfTheDayServiceImpl.java:47]: The cache doesn`t work from a method inside the same class.

Comment (vorfolomey) [reply core/src/main/java/greencity/service/impl/FactOfTheDayServiceImpl.java:47]: image

**Review by antonbakalets — APPROVED** [review]

**Review by forestik — APPROVED** [review]

**Review by olenapetryshak — APPROVED** [review]

**Review by Yura09 — APPROVED** [review]

**Pull request merged**